articles/node-js-vs-django/index.md

```diff
@@ -0,0 +1,83 @@
+Node.js and Django are technologies used in building versatile web and mobile applications. Both have an excellent user interface, are open, and do not require a license to operate, and therefore they have a vast number of users. Node.js is a framework that JavaScript uses on a client-side web application. Django works on [Python applications](https://www.javatpoint.com/python-applications) to help web developers build apps more quickly.
+
+This article will analyze these two popular technologies. The article will critically give each member the pros and cons to help the reader choose the best technology depending on the project needs.
+
+### Node.js
+
+It is an open-source JavaScript run time. The first version was released in the [year 2009](https://nodejs.dev/learn/a-brief-history-of-nodejs). Node.js is based on the following programming languages; C, C++, and JavaScript. Node.js application is on the client-side browser. It provides scalability and concurrency.
+
+#### Basic features of Node.js
+
+1. Simplicity – It is easy to learn and use and especially with JavaScript knowledge.
+2. Open-source language – It does not require installation fees and is free to use and readily available to the public.
+3. Object-oriented – It can model real-world data rather than functions and codes.
+4. It is swift to execute codes.
+5. A lot of community tutorials, resources, and shared codes are available.
+
+### Django
+
+It is an open-source Python web framework primarily designed to ensure high-speed web development. It is easy to learn for those knowledgeable in Python. Django has strict compliance with the rules and formalized scenarios for project implementation. Django provides accuracy in coding, a straight-forward app architecture, fewer errors, and bugs in the output.
+
+#### Basic features of Django
+
+1. Simplicity – Especially with the Python language, Django is easy to learn, use, and implement.
+2. It is an open-source language – No fees for installation and usage. It is available and freely accessible to the public.
+3. High-level language – It simplifies computer syntax into a language the user can easily understand.
```

Comment:
Isn't JS also a high-level language?

Reply:
Yes it is, I have included that feature, thank you.